trace/MANIFEST.in

```diff
@@ -1,5 +1,6 @@
 include README.rst LICENSE
-global-include *.json
+recursive-include unit_tests *
```

Comment:
I just saw this line appears in the MANIFEST.in file of all other packages in this repo, so I added it here... Not sure why the name is unit_tests.